packages/@uppy/utils/src/Translator.ts

```diff
@@ -6,10 +6,19 @@ export interface Locale<T extends number = number> {
   pluralize: (n: number) => T
 }
 
+export type OptionalPluralizeLocale<T extends number = number> =
+  | (Omit<Locale<T>, 'pluralize'> & {
+      pluralize?: (n: number) => T
+    })
+  | undefined
+
+// eslint-disable-next-line no-use-before-define
+export type I18n = Translator['translate']
+
 type Options = {
   smart_count?: number
 } & {
-  [key: string]: string | number
+  [key: string]: string | number | undefined
```

Comment:
`undefined` is not a valid value?
```suggestion
  [key: string]: string | number
```

Reply:
It creates two new errors, but I'm not able to see which ones (because there are too many other errors). Still applying it as I believe the types are more "correct"/close to the actual implementation with my suggestion